from core.shared.exceptions import NotFoundException


class OperationNotFoundException(NotFoundException):
    pass
